### PR TITLE
use docker to build for all platforms

### PR DESCRIPTION
**Pull request by ahdinosaur**
https://www.electron.build/multi-platform-build#docker

however only Mac can build for Mac because Apple code-signing reasons.

so on Linux i'm able to build for Linux and Windows.

### CODE REVIEW

**Comment by cblgh** [comment]
ohhh thanks @ahdinosaur! ✨

**Comment by ahdinosaur** [comment]
credit to https://github.com/ssbc/patchwork-builder for inspiration

**Comment by nikolaiwarner** [comment]
Great work! Thanks!!! 🍩

File: package.json [hunk 1]
```diff
@@ -61,6 +62,14 @@
   },
   "build": {
     "appId": "club.cabal.desktop",
+    "protocols": [
+      {
+        "name": "Dat Link",
+        "schemes": [
+          "dat"
```
Comment (noffle) [inline package.json]: `"cabal"`?

**Comment by noffle** [comment]
Does anyone know what is the next step toward automating these builds on releases?

**Pull request merged**